scripts/ingest_folder.py

```diff
@@ -69,8 +69,10 @@ def _do_ingest(changed_path: Path) -> None:
             logger.info(f"Started ingesting {changed_path}")
             ingest_service.ingest(changed_path.name, changed_path)
             logger.info(f"Completed ingesting {changed_path}")
-    except Exception as e:
-        logger.error(f"Failed to ingest document: {changed_path}. Error: {e}")
+    except Exception:
+        logger.exception(
```

Comment:
`error` -> `exception` to display the stack trace when folks are ingesting files using this script.

It will be helpful to debug folks (that seems to have `llama_index` complaining about missing packages while it is installed